rewatch/src/project_context.rs

```diff
@@ -0,0 +1,288 @@
+use crate::build::packages;
+use crate::config::Config;
+use crate::helpers;
+use ahash::AHashSet;
+use anyhow::Result;
+use anyhow::anyhow;
+use log::debug;
+use std::fmt;
+use std::path::{Path, PathBuf};
+
+type RescriptJsonPath = PathBuf;
```

Comment:
I would name this `ConfigPath` or something like that for now - or rename everything related to the config file to be `rescript` - I think there are still some 'old' names around

Reply:
(just for consistency)